scripts/install

```diff
@@ -88,12 +98,16 @@ ynh_script_progression --message="Setting up source files..." --weight=64
 
 ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 
-mkdir -p $final_path
-
-pushd $final_path
-
-PIPENV_VENV_IN_PROJECT="enabled" PIPENV_SKIP_LOCK=true  ynh_exec_warn_less python3 -m pipenv install jupyterlab==$jupyterlab_version jupyterhub notebook jupyterhub-ldapauthenticator pyzmq --three
-
+# Set permissions to app files
+mkdir -p "$final_path/.venv"
+chown -R $app:$app "$final_path"
+chmod -R g=u,g-w,o-rwx "$final_path"
+setfacl -nR -m g:$app.main:rx -m d:g:$app.main:rx "$final_path/.venv"
+setfacl -n -m g:$app.main:x "$final_path"
```

Comment:
Uh this `$app.main` is weird, is this really what you meant ? Isnt this related to the permission bug being discussed ?

Reply:
```suggestion
setfacl -nR -m g:$app:rx -m d:g:$app:rx "$final_path/.venv"
setfacl -n -m g:$app:x "$final_path"
```

Bumpitybump, naively I think $app should be enough, but I don't know what thoughts were being the original $app.main :s